pub/index.html

```diff
@@ -4,40 +4,154 @@
 <!--[if gt IE 8]><!--> <html class="no-js"> <!--<![endif]-->
     <head>
-        <link href='http://fonts.googleapis.com/css?family=Oswald:400,300,700|Open+Sans:400italic,700,300,600,400' rel='stylesheet' type='text/css'>
         <meta charset="utf-8">
-        <meta http-equiv="X-UA-Compatible" content="IE=edge,chrome=1">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
         <title></title>
         <meta name="description" content="">
         <meta name="viewport" content="width=device-width, initial-scale=1">
-        
+
+        <!-- Place favicon.ico and apple-touch-icon.png in the root directory -->
+        <link href='http://fonts.googleapis.com/css?family=Oswald:400,300,700|Open+Sans:400,300,600,700' rel='stylesheet' type='text/css'>
         <link href="//maxcdn.bootstrapcdn.com/font-awesome/4.2.0/css/font-awesome.min.css" rel="stylesheet">
-        <link href='http://fonts.googleapis.com/css?family=Lato:100,400,700' rel='stylesheet' type='text/css'>
-        <link rel="stylesheet" href="css/normalize.min.css">
+        <link rel="stylesheet" href="css/normalize.css">
         <link rel="stylesheet" href="css/main.css">
-
         <script src="js/vendor/modernizr-2.6.2.min.js"></script>
     </head>
     <body>
-        <div class="container">
-            <ol class="timeline">
-                <li class="right-content">
-                    <time class="left-content">
-                        <small>1/14/2015</small>
-                        <br>
-                        <big>18:30</big>
+        <!--[if lt IE 7]>
+            <p class="browsehappy">You are using an <strong>outdated</strong> browser. Please <a href="http://browsehappy.com/">upgrade your browser</a> to improve your experience.</p>
+        <![endif]-->
+
+        <!-- Add your site or application content here -->
+        <header class="banner">
+            <nav class="main-nav">
+                <a href="#">About</a>
+                <a href="#">Search <icon class="fa fa-search fa-flip-horizontal"></icon></a>
+            </nav>
+            <div class="logo-container">
+                <img src="img/cat-logo.png">
+                <p class="hidden-mobile">Cat &amp; Kitten Co. Blog</p>
+            </div>
+            <div class="headline">
+                <h1>Cat &amp; Kitten Co.</h1>
+                <p>An Ironyard Academy Project Est. Summer 2014</p>
+            </div>
+        </header>
+        <div class="main-container">
+            <section class="content">
+                <div class="text-container">
+                    <h2>I like big cats and i can not lie</h2>
+                    <time datetime="2015-07-09T19:22Z-4">
+                    <figure class="aside-image">
+                        <img src="http://lorempixel.com/output/cats-q-c-100-100-7.jpg">
+                        <figcaption>Cattie Cat</figcaption>
+                    </figure>   
+                    <small>on Monday, June 7th 9:22pm</small>
```

Comment:
This should really be the only contents of the `<time>` tag... And I've created a monster by showing you that `<figure>` tag. Girl, you're in _love_! 

While there's nothing wrong with your usage, better to provide the author information within `<address class="author">` or `<a href rel="author">` that links to a user profile, etc.